deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/__init__.py

```diff
@@ -67,6 +68,15 @@
     }
 ]
 
+SUSE_SYSTEM_PACKAGE = {
+    'SLES15': [{
+                "name": "sle-module-basesystem-release",
+                "version": "15.2",
+                "format": "rpm",
+                "vendor": "SUSE LLC <https://www.suse.com/>"
+              }]
+}
+
```

Comment:
Fix indent

Reply:
When running pycodestyle, it does not show an indentation error here. Also, this has the same indentation pattern as the `CUSTOM_PACKAGES` in the same file, and the `SYSTEM_DATA` dictionary in `mocking/__init__.py`. and 

Please inform on  how to handle and if change is needed, would it also

Reply:
The correct format would be as follows:

```
SUSE_SYSTEM_PACKAGE = {
    'SLES15': [
        {
            "name": "sle-module-basesystem-release",
            "version": "15.2",
            "format": "rpm",
            "vendor": "SUSE LLC <https://www.suse.com/>"
        }
    ]
}
```

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2847/commits/022e2e38dcd8df4814602add1fb62c5acf8a6a0a